modules/gapi/samples/face_detection.cpp

```diff
@@ -21,18 +21,19 @@
 const std::string about =
 "This is an OpenCV-based version of OMZ MTCNN Face Detection example";
 const std::string keys =
-"{ h help     |                           | Print this help message }"
-"{ input      |                           | Path to the input video file }"
-"{ mtcnnpm    | mtcnn-p.xml               | Path to OpenVINO MTCNN P (Proposal) detection model (.xml)}"
-"{ mtcnnpd    | CPU                       | Target device for the MTCNN P (e.g. CPU, GPU, VPU, ...) }"
-"{ mtcnnrm    | mtcnn-r.xml               | Path to OpenVINO MTCNN R (Refinement) detection model (.xml)}"
-"{ mtcnnrd    | CPU                       | Target device for the MTCNN R (e.g. CPU, GPU, VPU, ...) }"
-"{ mtcnnom    | mtcnn-o.xml               | Path to OpenVINO MTCNN O (Output) detection model (.xml)}"
-"{ mtcnnod    | CPU                       | Target device for the MTCNN O (e.g. CPU, GPU, VPU, ...) }"
-"{ thrp       | 0.6                       | MTCNN P confidence threshold}"
-"{ thrr       | 0.7                       | MTCNN R confidence threshold}"
-"{ thro       | 0.7                       | MTCNN O confidence threshold}"
-"{ half_scale | false                     | MTCNN P use half scale pyramid}"
+"{ h help           |                           | Print this help message }"
+"{ input            |                           | Path to the input video file }"
+"{ mtcnnpm          | mtcnn-p.xml               | Path to OpenVINO MTCNN P (Proposal) detection model (.xml)}"
+"{ mtcnnpd          | CPU                       | Target device for the MTCNN P (e.g. CPU, GPU, VPU, ...) }"
+"{ mtcnnrm          | mtcnn-r.xml               | Path to OpenVINO MTCNN R (Refinement) detection model (.xml)}"
+"{ mtcnnrd          | CPU                       | Target device for the MTCNN R (e.g. CPU, GPU, VPU, ...) }"
+"{ mtcnnom          | mtcnn-o.xml               | Path to OpenVINO MTCNN O (Output) detection model (.xml)}"
+"{ mtcnnod          | CPU                       | Target device for the MTCNN O (e.g. CPU, GPU, VPU, ...) }"
+"{ thrp             | 0.6                       | MTCNN P confidence threshold}"
+"{ thrr             | 0.7                       | MTCNN R confidence threshold}"
+"{ thro             | 0.7                       | MTCNN O confidence threshold}"
+"{ half_scale       | false                     | MTCNN P use half scale pyramid}"
+"{ queue_capacity   | 1                         | Streaming executor queue capacity. Calculated automaticaly if 0}"
```

Comment:
M B `-1` will be value for automatic calculation and it will be asserted for zero that `queue capacity != 0`.

Reply:
https://github.com/opencv/opencv/blob/master/modules/gapi/src/executor/gstreamingexecutor.cpp#L1139

Reply:
queue capacity = 0 case is documented in the sample help string. The sample will never try to provide compile args with zero queue depth by design.